test/model_factory_test.py

```diff
@@ -55,6 +55,8 @@ def test_cfg_set_parsing(self):
 
         self.assertEqual(first_elem_from_fac.raw, first_element.raw)
 
+    ### Tests for _cfg_element_names and _cfg_elements in ConfigFactory
+
     def test_cfg_element_names(self):
```

Comment:
this may be the method name you test; it would however be nice to be more expressive; e.g. "test_enumerate_element_names" or sth. similar

Reply:
other naming proposal: test_cfg_element_names_should_return_all_element_names